utility/src/main/java/org/oppia/android/util/extensions/ContextExtensions.kt

```diff
@@ -9,6 +9,11 @@ fun Context.getVersionName(): String {
   return this.packageManager.getPackageInfo(this.packageName, /* flags=*/ 0).versionName
 }
 
+/** Returns the current app version code. */
+fun Context.getVersionCode(): Int {
+  return this.packageManager.getPackageInfo(this.packageName, /* flags=*/ 0).versionCode
```

Comment:
```suggestion
  return this.packageManager.getPackageInfo(this.packageName, /* flags= */ 0).versionCode
```

Maybe fix the one above, too, for an opportunistic fix.

Reply:
Done